Plugins/Player/Player.cpp

```diff
@@ -1428,4 +1430,30 @@ ArgumentStack Player::SetCreatureNameOverride(ArgumentStack&& args)
     return Services::Events::Arguments();
 }
 
+ArgumentStack Player::FloatingTextStringOnCreature(ArgumentStack&& args)
+{
+    if (auto *pPlayer = player(args))
+    {
+        auto oidCreature = Services::Events::ExtractArgument<ObjectID>(args);
+          ASSERT_OR_THROW(oidCreature != Constants::OBJECT_INVALID);
```

Comment:
Is there a reason this is indented by 2 extra? I un-indented this in my MR.

Reply:
This half-indent was used for some of these ASSERT_OR_THROW calls to make it more obvious it is relevant to the line above. Originally, the asserts were in the same line as extract, after the `;`, but that was unwieldy so this was done.

I don't really feel strongly either way. It _does_ look slightly nicer to me, but it is also another thing to remember and it's just not worth the cognitive load to enforce.